drivers/hmis/app/jobs/hmis/match_candidates_job.rb

```diff
@@ -25,14 +25,16 @@ def perform(**args)
     # @param opportunity_ids [Array] an array of opportunity IDs to build candidate pools for.
     # If nil, builds pools for all active Opportunities
     def _perform(opportunity_ids: nil)
+      raise unless Hmis::Ce.configuration.enabled?
+
       opportunities = opportunity_ids ? Hmis::Ce::Opportunity.where(id: opportunity_ids) : Hmis::Ce::Opportunity.active
       log("Building candidate pools for #{opportunities.count} opportunities")
       Hmis::Ce::Match::CandidatePoolBuilder.new(opportunities).perform
 
       # Find candidate pools for these opportunities
       candidate_pool_ids = opportunities.reload.pluck(:candidate_pool_id).compact.uniq
       candidate_pool_scope = Hmis::Ce::Match::CandidatePool.where(id: candidate_pool_ids)
-      client_scope = Hmis::Hud::Client.hmis
+      client_scope = ::GrdaWarehouse::Hud::Client.where(data_source_id: ::GrdaWarehouse::DataSource.destination_data_source_ids)
```

Comment:
I think we can use the existing scope
```suggestion
      client_scope = ::GrdaWarehouse::Hud::Client.destination
```